src/verbosity.jl

```diff
@@ -0,0 +1,89 @@
+function restore_callsite_source_position!(expr, src)
+    # used to fix the logging source file + line
+    # since we're lowering our verbose logging macros to the
+    # Logging.jl macros; otherwise, they would always report this (verbosity.jl)
+    # file as the logging callsite
+    expr.args[1].args[2] = src
+    return expr
+end
+
+vlogmacrodocs = """
+    @debugv N msg args...
+    @infov N msg args...
+    @warnv N msg args...
+    @errorv N msg args...
```

Comment:
Should these be dropped now? They are just slightly shorter versions of `@logmsg LogLever(..) args...`, and not really related to verbosity.

Reply:
Non-starter. They're considerably more legible and indicate what they're for vs. `@logmsg Loglevel(N) args...`. In my mind, they _are_ still related to verbosity because they're modifying the level by `N` to increase that level.

Then again, this was part of why I liked having verbosity as a separate arg/dimensions _in addition to_ level, because I don't think viewing log levels as a "verbosity" spectrum is quite accurate. I do think it's "fine" and works since the log levels themselves are so far apart and so "taking" the few levels right below each level for "verbosity" should work well in practice, but I'm really viewing verbosity as kind of a distinct thing from the more categorical log level.

Reply:
I am happy to keep these. I broadly agree.